### PR TITLE
[VISUALIZATION] Apply code-checks/format with misc-definitions-in-headers

### PR DESCRIPTION
**Pull request by smuzaffar**
code-checks/format applied on full cmssw

### CODE REVIEW

File: Fireworks/Core/src/CmsShowMain.cc [hunk 1]
```diff
@@ -160,33 +160,120 @@ CmsShowMain::CmsShowMain(int argc, char* argv[])
...
+                                                                                                                 "with "
+                                                                                                                 "bad "
+                                                                                                                 "hardw"
+                                                                                                                 "are "
+                                                                                                                 "drive"
+                                                                                                                 "r"
+                                                                                                                 "s")(kExpertCommandOpt,
+                                                                                                                      "Enable PF user plugins.")(kHelpCommandOpt,
+                                                                                                                                                 "Display help message");
```
Comment (makortel) [inline Fireworks/Core/src/CmsShowMain.cc:222]: I'd say -1 though...

**Comment by alja** [comment]
@makortel
@smuzaffar 
Is there a way to tell formatter to leave out some code?

**Comment by makortel** [comment]
There is, but we have the same issue on many exetutables in FWCore, so let's figure out a common solution first.

File: Fireworks/Core/src/CmsShowMain.cc (hunk 1 above)
Comment (smuzaffar) [reply Fireworks/Core/src/CmsShowMain.cc:222]: agree, we should add 
```
// clang-format off

//// clang-format on
```

**Comment by smuzaffar** [comment]
@makortel , I think there is no automatic solution. We do want clang-format to format the code and wrap the lines to 120 chars. I would suggest that so cases like this and few in Core we just wrap the section with 
```
// clang-format off

// clang-format on
```

**Comment by makortel** [comment]
> @makortel , I think there is no automatic solution. We do want clang-format to format the code and wrap the lines to 120 chars. I would suggest that so cases like this and few in Core we just wrap the section with
> 
> ```
> // clang-format off
> 
> // clang-format on
> ```

Ok (pretty much what I was afraid of). How do we proceed in practice? Should we format these sections manually now (that they're not that awful yet) and repeat the automatic formatting, or merge these PRs and format manually afterwards?

**Comment by smuzaffar** [comment]
@makortel ,  yes I can fix this PR and one for core ( https://github.com/cms-sw/cmssw/pull/34036 ) and revert the formatting changes ( where it is really awful) and add `// clang-format off/on`  around those sections.

**Comment by makortel** [comment]
Thanks @smuzaffar!

**Comment by smuzaffar** [comment]
please test

**Comment by qliphy** [comment]
+1

**Pull request merged**